solr/bin/solr

```diff
@@ -716,7 +716,11 @@ function solr_pid_by_port() {
 # extract the value of the -Djetty.port parameter from a running Solr process
 function jetty_port() {
   SOLR_PID="$1"
-  SOLR_PROC=$(ps -o command='' -p "$SOLR_PID" | grep start\.jar | grep jetty\.port)
+  if [ "OS400" == "$THIS_OS" ]; then
+    SOLR_PROC=`ps -f -p "$SOLR_PID" | grep start\.jar | grep jetty\.port`
```

Comment:
Please stick to the `$(foo)` syntax

Reply:
Sorry for the delay. Fixed at latest commit/force-push. Thanks!